src/compiler/crystal/tools/formatter.cr

```diff
@@ -2060,14 +2060,14 @@ module Crystal
       end
 
       # This is the case of an enum member
-      if node.name[0].ascii_uppercase? && @token.type == :","
-        write ","
```

Comment:
I'm not sure what the usual policy for the formatter is, but might it be possible to tell it to chew up old code with commas and re-format it, rather than outright failing?

Reply:
Good idea! We've done it in the past. But for that the parser will still have to support commas, or at least the parser used in the formatter (could be toggled with a flag). I like that idea.

Reply:
That would be similar to an autocorrection feature for `check`, but implemented in the formatter.
I don't think it's urgent to completely abolish commas right away, so formatting them to semicolons seems like a good idea :+1: